### PR TITLE
More tf docs

### PR DESCRIPTION
**Pull request by rrrkharse**
- Fix broken Cognito doc ref ([Failing build](https://github.com/awslabs/kubeflow-manifests/actions/runs/3097938991/jobs/5015216545))
- Format Terraform docs with numbering

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by surajkota — CHANGES_REQUESTED** [review]

File: website/content/en/docs/deployment/rds-s3/guide-terraform.md [hunk 1]
```diff
@@ -39,39 +39,43 @@ pwd
...
-
-# The below values are set to make cleanup easier but are not recommended for production
-deletion_protection="false"
-secret_recovery_window_in_days="0"
-force_destroy_s3_bucket="true"
-EOF
-```
+1. Create an IAM user to use with the Minio Client
+
+    [Create an IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html#id_users_create_cliwpsapi) with permissions to get bucket locations and allow read and write access to objects in an S3 bucket where you want to store the Kubeflow artifacts. Take note of the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY of the IAM user that you created to use in the following step, which will be referenced as TF_VAR_minio_aws_access_key_id and TF_VAR_minio_aws_secret_access_key respectively.
```
[the remark this answers is not in the document]
Comment (rrrkharse) [reply website/content/en/docs/deployment/rds-s3/guide-terraform.md]: done

File: website/content/en/docs/deployment/cognito/guide-terraform.md [hunk 2]
```diff
@@ -40,35 +40,41 @@ pwd
...
-# Name of the cluster to create
-export CLUSTER_NAME=
-# Name of an existing Route53 root domain (e.g. example.com)
-export ROOT_DOMAIN=
-# Name of the subdomain to create (e.g. platform.example.com)
-export SUBDOMAIN=
-# Name of the cognito user pool to create
-export USER_POOL_NAME=
-```
+1. The provided Terraform stack will create a subdomain manually, however you can disable this behavior and provide your own subdomain.
```
[the remark this answers is not in the document]
Comment (rrrkharse) [reply website/content/en/docs/deployment/cognito/guide-terraform.md]: done

File: website/content/en/docs/deployment/cognito/guide-terraform.md [hunk 3]
```diff
@@ -40,35 +40,41 @@ pwd
 
 ### Configure
 
-Create a root domain manually (e.g. not through Terraform.) To create a domain as the root domain through Route53 follow the steps [here](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingHostedZone.html).
+1. Create a root domain
 
-To create a subdomain manually as well follow the steps [here]({{< ref "../../add-ons/load-balancer/guide/#create-domain-and-certificates" >}}).
+    Create a root domain manually (e.g. through the AWS console, a different Terraform stack, etc.) To create a domain as the root domain through Route53 follow the steps [here](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingHostedZone.html).
```
[the remark this answers is not in the document]
Comment (rrrkharse) [reply website/content/en/docs/deployment/cognito/guide-terraform.md]: done

**Review by surajkota — APPROVED** [review]
Cleann!!

**Pull request merged**